install-usercss.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Loading...</title>
+  <link rel="stylesheet" href="/install-usercss/install-usercss.css">
+  <script src="/js/messaging.js"></script>
+  <script src="/js/prefs.js"></script>
+  <script src="/js/dom.js"></script>
+  <script src="/js/localization.js"></script>
+  <script src="/vendor/node-semver/semver.js"></script>
+
+  <!-- FIXME: do we need all of these? -->
```

Comment:
Why not?

Reply:
They are copy-paste from `edit.html`. I'm not sure if all the addons are needed since we only display a readonly editor on install page.

Reply:
These keymaps have different bindings for cursor movement so I guess we need them.